test/cmp_vfy_test.c

```diff
@@ -336,8 +336,9 @@ static int test_validate_msg_signature_sender_cert_extracert(void)
             || !add_trusted(fixture->cmp_ctx, instaca_cert)) {
         tear_down(fixture);
         fixture = NULL;
+    } else {
+        fixture->cert = sk_X509_value(fixture->msg->extraCerts, 1); /* Insta CA */
     }
-    fixture->cert = sk_X509_value(fixture->msg->extraCerts, 1); /* Insta CA */
     EXECUTE_TEST(execute_validate_msg_test, tear_down);
```

Comment:
Do we want to execute the test if we couldn't create the fixture required?

Reply:
The macro EXECUTE_TEST checks fixture for being non-NULL. This is done also in other testcases.